utility/MK3SGL.h

```diff
@@ -186,6 +187,12 @@ class MK3SGL: public BasePeripheral, public Scriptable, private IKeyClient
 			m_bSDCard = {true},
 			m_bPrintSurface = {true};
 
+		float xrotate=0.f;
```

Comment:
Please follow the established variable naming conventions with the appropriate Hungarian notation prefixes.